.github/workflows/ci.yml

```diff
@@ -0,0 +1,69 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request: ~
+
+jobs:
+  # derived from https://dev.to/felipetofoli/github-actions-for-net-full-framework-build-and-test-299h
+  NET_Tests:
+    runs-on: windows-2019
+    strategy:
+      matrix:
+        dotnet-version: [ '35', '40' ]
+    steps:
+      - uses: actions/checkout@v1
+      - name: Setup variables
+        id: test_file
+        run: echo "::set-output name=test_file::EasyPost.Tests.Net${{ matrix.dotnet-version }}"
+      - name: Setup MSBuild
+        if: always()
+        uses: microsoft/setup-msbuild@v1.1
+      - name: Setup Nuget
+        uses: NuGet/setup-nuget@v1.0.5
+      - name: Setup VSTest
+        uses: darenm/Setup-VSTest@v1
```

Comment:
What does this action do? It's not clear from its README what its purpose is. Looks like below there is a command invoked from this action?

Reply:
Setup Nuget installs nuget.exe, which is needed during the `nuget restore` action later.

Reply:
My question above was attached to the `darenm/Setup-VSTest@` line, what does that one do?

Reply:
Ah, yes, this imports `vstest.console.exe` (Visual Studio's CLI testing program), which is used to execute the test DLLs later.